VERSION

```diff
@@ -14,7 +14,7 @@
   "miners": {
     "version": "0.2.0",
     "build": "",
-    "date": "2016-06-27",
+    "date": "2016-06-28",
```

Comment:
You should be happy I got the year right, I'm still getting that wrong on checks :)